docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx

```diff
@@ -3,19 +3,58 @@ import CLI from './quick-start-pull-cli.mdx';
 import Callout from './credentials-provider-callout.mdx';
 import { Alert, Tabs } from '@aws-amplify/ui-react';
 import AppDirectoryAlert from '@/components/AppDirectoryAlert';
-import { InstallScripts } from '@/components/InstallScripts';
+import { InstallScripts, TerminalCommand } from '@/components/InstallScripts';
 import Liveness from './liveness-service-setup.mdx';
 
 There are three methods to setup the Amazon Cognito resources needed for Amplify Auth. The most common is the Amplify CLI create flow which asks a series of questions and will configure both a User Pool and Identity Pool automatically. The second option is the Amplify CLI import flow which adds an existing Cognito resource into Amplify. Finally the third is to reuse or create a Cognito Identity Pool manually and to add it into your application.
 
 <CognitoCallout />
 
-<Tabs.Container defaultValue="Amplify CLI">
+<Tabs.Container defaultValue="Gen 2">
   <Tabs.List>
+    <Tabs.Item value="Gen 2">Gen 2</Tabs.Item>
     <Tabs.Item value="Amplify CLI">Amplify CLI</Tabs.Item>
     <Tabs.Item value="Amplify CLI Import">Amplify CLI Import</Tabs.Item>
```

Comment:
I realize this should have some other naming like `Amplify Gen 2` vs `Amplify CLI v1` or something? Just not sure what the names should be

Reply:
Think we have been referring to CLI built apps as Gen1, probably worth checking how the main docs is handling the change